### PR TITLE
LPS-162453

### PR DESCRIPTION
**Pull request by leticia-maciel**

### CODE REVIEW

**Comment by leticia-maciel** [comment]
ci:test:sf

**Comment by rodrigopaulino** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/function/CallFunction.java [hunk 1]
```diff
@@ -189,13 +189,15 @@ protected void setDDMFormFieldValues(
 
 		for (Map.Entry<String, String> entry : resultMap.entrySet()) {
 			String outputName = entry.getValue();
+			String ddmFormFieldName = entry.getKey();
 
 			if (!ddmDataProviderResponse.hasOutput(outputName)) {
+				setDDMFormFieldOptions(
```
Comment (rodrigopaulino) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/function/CallFunction.java:195]: Ideally, we should be certain that the field uses options. Take a numeric field being used as data provider output as an example.

**Review by rodrigopaulino — CHANGES_REQUESTED** [review]
Hey, @leticia-maciel.
Nice find and your fix works for your purpose and the customer's, although I think there's a more robust solution for this that you can find described below.
Could you please address it and resend the PR?
Thanks a lot for contributing.

**Comment by rodrigopaulino** [comment]
ci:close

**Pull request closed without merging**